contentcuration/contentcuration/management/commands/loadconstants.py

```diff
@@ -154,15 +154,7 @@
         "pk": "extension",
         "fields": {
             "extension": file_formats.VTT,
-            "mimetype" : file_formats.VTT_MIMETYPE,
-        },
-    },
-    {
-        "model": models.FileFormat,
-        "pk": "extension",
-        "fields": {
-            "extension": file_formats.SRT,
-            "mimetype" : file_formats.SRT_MIMETYPE,
+            "mimetype" : ".vtt",
```

Comment:
Don't allow .srt files on subtitles, use .vtt as not all systems recognize vtts as the same mimetype